### PR TITLE
[celestica] change to use lm75 standard driver

### PR DESCRIPTION
**Pull request by pwisutti**
**- What I did**
- Change the lm75 driver to standard driver for kernel version 4.9.
- Update init script for creating lm75 devices because the driver's name is changed.
- 
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by lguohan** [comment]
you can remove the lm75 driver since the kernel image has the driver, you do not need to build it in your module.

**Review by lguohan — CHANGES_REQUESTED** [review]
just remove the lm75 driver from your module.

**Comment by pwisutti** [comment]
I've already removed lm75 driver from the module and also configured the init script to probe lm75 driver.

File: platform/broadcom/sonic-platform-modules-cel/dx010/modules/Makefile [hunk 1]
```diff
@@ -1,6 +1 @@
 obj-m := dx010_cpld.o mc24lc64t.o emc2305.o dx010_wdt.o leds-dx010.o lm75.o
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/dx010/modules/Makefile]: you still have lm75.o here?

Comment (lguohan) [reply platform/broadcom/sonic-platform-modules-cel/dx010/modules/Makefile]: can you make sure your dx010 module can compile as it is?

Comment (pwisutti) [reply platform/broadcom/sonic-platform-modules-cel/dx010/modules/Makefile]: Yes, it should be removed. I've fixed

**Review by lguohan — APPROVED** [review]

**Pull request merged**